src/Model.jl

```diff
@@ -218,25 +218,38 @@ end
     Model(model; [lattice, positions, atoms, kwargs...])
     Model{T}(model; [lattice, positions, atoms, kwargs...])
 
-Construct an identical model to `model` with the option to change some of the contained
-parameters. This constructor is useful for changing the data type in the model
-or for changing `lattice` or `positions` in geometry/lattice optimisations.
+Construct an identical model to `model` with the option to change some of
+the contained parameters. This constructor is useful for changing the data
+type in the model or for changing `lattice` or `positions` in
+geometry/lattice optimisations.
+
+!!! warning
```

Comment:
We should definitely do that by default and be able to turn it off via a flag

Reply:
Agree. But it's breaking and more work than what it looks (due to the magnetic moments), so let's do it later.